docs/reference/api.mdx

```diff
@@ -10,9 +10,11 @@ import EndpointList from "@site/src/components/EndpointList/EndpointList";
 
 If you are a developer we highly recommend signing up for a free private
 endpoint through [AllThatNode](https://www.allthatnode.com/pricing.dsrv),
-[BlockPi](https://blockpi.io/pricing), [Infstones](https://infstones.com/fast-api),
-or one of our other RPC partners. The private endpoints have higher throughput,
-less restrictions, and will provide a more consistent experience than the public RPCs.
+[BlockPi](https://blockpi.io/pricing),
+[Infstones](https://infstones.com/fast-api),
+[Blast API](https://blastapi.io/pricing) or one of our other RPC partners. The
```

Comment:
https://blastapi.io/chains/zetachain may be a better link (IDT the other current providers have a similar dedicated page)

Reply:
a257b1d